ci/README.md

```diff
@@ -0,0 +1,129 @@
+# CI tools
+
+Continuous Integration tools used by the `rust-bitcoin` org.
+
+#### Table Of Contents
+
+- [Repository](#repository)
+  * [Lock files](#lock-files)
+  * [Crates](#crates)
+  * [Pinning](#dependency-pinning)
+- [Workflows](#workflows)
+- [get_matrix.sh](#get_matrix.sh)
+- [test_vars.sh](#test_vars.sh)
+- [run_task.sh](#run_task.sh)
+  * [Environment variables](#environment-variables)
+  * [Integration tests](#integration-tests)
+  * [Crate specific tests](#crate-specific-tests)
+  * [Fuzzing](#fuzzing)
+
+## Repository
+
+The `run_task.sh` script expects a few things to be present in all repositories.
+
+- lock files: `Cargo-recent.lock` and `Cargo-minimal.lock`
+- crate declarations: `contrib/crates.sh`
+- pinning: `contrib/pin.sh`
+
+## Lock files
+
+All repositories MUST include a minimal and recent lock file:
```

Comment:
```suggestion
All repositories MUST include a minimal and a recent lock file:
```

It is two right? Not one that is minimal and recent?

Reply:
Lol you got me, and English is even my first language :)